### PR TITLE
MAPI-1421 - enabled MacWire in notification sub-project

### PR DESCRIPTION
**Pull request by agulowaty**
* macwire dep added
* reverted application.conf files in n10n - Application loaders need to be defined there

cc @davidfurey @alexduf

### CODE REVIEW

File: notification/app/notification/NotificationApplicationLoader.scala [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+trait AzureHubComponents {
+  self: ConfigurationComponents with AhcWSComponents with ExecutionEnv =>
+
+  import com.amazonaws.regions.Regions.EU_WEST_1
+  import aws.AsyncDynamo
+
+  lazy val hubClient = {
+    val hubConnection = NotificationHubConnection(
```
Comment (alexduf) [inline notification/app/notification/NotificationApplicationLoader.scala]: It may be worth doing that in another PR, but maybe NotificationHubConnection should directly recive the appConfig as a parameter. That would allow you to write it like that: ```val hubConnection = wire[NotificationHubConnection]```

File: notification/app/notification/services/frontend/FrontendAlerts.scala [hunk 2]
```diff
@@ -8,14 +10,15 @@ import play.api.Logger
...
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 import scalaz.{-\/, \/-, \/}
 import scalaz.syntax.either._
 import scalaz.syntax.std.option._
 
-class FrontendAlerts(config: FrontendAlertsConfig, wsClient: WSClient) extends NotificationSender {
+case class FrontendAlertsConfig(endpoint: URI, apiKey: String)
+
+class FrontendAlerts(config: FrontendAlertsConfig, wsClient: WSClient)(implicit val ec: ExecutionContext) extends NotificationSender {
```
Comment (alexduf) [inline notification/app/notification/services/frontend/FrontendAlerts.scala]: Nice, the least we use ```scala.concurrent.ExecutionContext.Implicits.global``` the better

File: notification/app/notification/NotificationApplicationLoader.scala (hunk 1 above)
Comment (agulowaty) [inline notification/app/notification/NotificationApplicationLoader.scala]: I'm not a fan of passing more data than it is actually needed as configuration parameter - this creates hidden dependency from Client to global AppConfig

**Comment by alexduf** [comment]
Great PR :+1: it's a clear improvement

File: notification/app/notification/NotificationApplicationLoader.scala (hunk 1 above)
Comment (alexduf) [inline notification/app/notification/NotificationApplicationLoader.scala]: Fair enough

**Comment by agulowaty** [comment]
Next steps: MacWire in registration and report. Thanks @alexduf
